src/integration/masternode/domain/entities/masternode.entity.ts

```diff
@@ -20,8 +20,8 @@ export class Masternode extends IEntity {
   @Column({ nullable: true })
   ownerWallet: string;
 
-  @Column({ type: 'integer', nullable: true })
-  timeLock: number;
+  @Column({ default: MasternodeTimeLock.NONE, nullable: false })
+  timeLock: MasternodeTimeLock;
```

Comment:
Do we really want to have a default time lock value?

Reply:
either we have a default value in the database, or a default value we can fallback to on masternode creation. I have thought that having a default value in the database is more obvious to changing it

Reply:
fixed in #42